react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -527,6 +531,36 @@ public void afterTextChanged(Editable s) {
         }
     }
 
+    private enum ReactAztecTextFormat {
+        H1("h1", AztecTextFormat.FORMAT_HEADING_1),
+        H2("h2", AztecTextFormat.FORMAT_HEADING_2),
+        H3("h3", AztecTextFormat.FORMAT_HEADING_3),
+        H4("h4", AztecTextFormat.FORMAT_HEADING_4),
+        H5("h5", AztecTextFormat.FORMAT_HEADING_5),
+        H6("h6", AztecTextFormat.FORMAT_HEADING_6);
```

Comment:
I suggest to add the PARA item here: `P("p", AztecTextFormat.FORMAT_PARAGRAPH);`

and maybe a comment for future reference, since it's not easy to understand why we need to toggle the formatting.

Reply:
Thanks for the comments @daniloercoli, I have made those changes.